backend/shelf-server/src/main/java/com/htec/shelfserver/service/UserService.java

```diff
@@ -153,4 +157,21 @@ public UserResponseModel updateUser(UserDTO userDTO) {
 
         return UserMapper.INSTANCE.userEntityToUserResponseModel(user);
     }
+
+    public UserResponseModel updateUserRole(Long id, Long roleId) {
+
+        UserEntity user = userRepository.findById(id).orElseThrow(ExceptionSupplier.recordNotFoundWithId);
+
+        if (user.getRole().getId().equals(roleId)) {
+            throw ExceptionSupplier.wrongRoleUpdate.get();
+        }
+
+        RoleEntity role = roleRepository.findById(roleId).orElseThrow(ExceptionSupplier.recordNotFoundWithId);
```

Comment:
new line after findById(roleId), try to be consistent :)

Reply:
Done